prater.tf

```diff
@@ -0,0 +1,120 @@
+locals {
+  /* Volumes */
+  prater_root_vol_size = 20
+  prater_data_vol_size = 150
+  prater_data_vol_type = "io1"
+  prater_data_vol_iops = 2500
+  /* Instances */
+  prater_large_instance_type = "z1d.large"
+}
+
+module "nimbus_nodes_prater_stable_large" {
+  source = "github.com/status-im/infra-tf-amazon-web-services"
+
+  name   = "stable-large"
+  env    = "nimbus"
+  stage  = "prater"
+  group  = "nimbus-prater-stable"
+  domain = var.domain
+
+  /* Scaling */
+  instance_type = local.prater_large_instance_type
+  root_vol_size = local.prater_root_vol_size
+  data_vol_size = local.prater_data_vol_size
+  data_vol_type = local.prater_data_vol_type
+  data_vol_iops = local.prater_data_vol_iops
+  host_count    = 5
+
+  /* Firewall */
+  open_udp_ports = local.nimbus_ports
+  open_tcp_ports = local.nimbus_ports
+
+  /* Plumbing */
+  vpc_id       = module.nimbus_network.vpc.id
+  subnet_id    = module.nimbus_network.subnets[0].id
+  secgroup_id  = module.nimbus_network.secgroup.id
+  keypair_name = aws_key_pair.jakubgs.key_name
+}
+
+module "nimbus_nodes_prater_testing_large" {
+  source = "github.com/status-im/infra-tf-amazon-web-services"
+
+  name   = "testing-large"
+  env    = "nimbus"
+  stage  = "prater"
+  group  = "nimbus-prater-testing"
+  domain = var.domain
+
+  /* Scaling */
+  instance_type = local.prater_large_instance_type
+  root_vol_size = local.prater_root_vol_size
+  data_vol_size = local.prater_data_vol_size
+  data_vol_type = local.prater_data_vol_type
+  data_vol_iops = local.prater_data_vol_iops
+  host_count    = 5
+
+  /* Firewall */
+  open_udp_ports = local.nimbus_ports
+  open_tcp_ports = local.nimbus_ports
+
+  /* Plumbing */
+  vpc_id       = module.nimbus_network.vpc.id
+  subnet_id    = module.nimbus_network.subnets[0].id
+  secgroup_id  = module.nimbus_network.secgroup.id
+  keypair_name = aws_key_pair.jakubgs.key_name
+}
+
+module "nimbus_nodes_prater_unstable_large" {
+  source = "github.com/status-im/infra-tf-amazon-web-services"
+
+  name   = "unstable-large"
+  env    = "nimbus"
+  stage  = "prater"
+  group  = "nimbus-prater-unstable"
+  domain = var.domain
+
+  /* Scaling */
+  instance_type = local.prater_large_instance_type
+  root_vol_size = local.prater_root_vol_size
+  data_vol_size = local.prater_data_vol_size
+  data_vol_type = local.prater_data_vol_type
+  data_vol_iops = local.prater_data_vol_iops
+  host_count    = 5
+
+  /* Firewall */
+  open_udp_ports = local.nimbus_ports
+  open_tcp_ports = local.nimbus_ports
+
+  /* Plumbing */
+  vpc_id       = module.nimbus_network.vpc.id
+  subnet_id    = module.nimbus_network.subnets[0].id
+  secgroup_id  = module.nimbus_network.secgroup.id
+  keypair_name = aws_key_pair.jakubgs.key_name
+}
+
+module "nimbus_nodes_prater_unstable_libp2p_large" {
+  source = "github.com/status-im/infra-tf-amazon-web-services"
+
+  name   = "unstable-libp2p-large"
+  env    = "nimbus"
+  stage  = "prater"
+  group  = "nimbus-prater-unstable-libp2p"
```

Comment:
The `unstable` part was there only because I had to give the new host a new name that won't clash with the old one.

We could drop it.